ci/Jenkinsfiles/export.groovy

```diff
@@ -255,14 +255,21 @@ pipeline {
                       --namespace=${PREVIEW_NAMESPACE}
 
                     ${curlCommand} ${explorerUrl} --output home.html
+                  """
+                }
+                retry (2) {
+                  sh """
                     ${curlCommand} -d 'name=${params.SNAPSHOT_NAME}' -d 'version=${NUXEO_IMAGE_VERSION}' -H 'Accept: text/plain' ${explorerUrl}/save
                     ${curlCommand} ${explorerUrl} --output home_after_save.html
-                    ${curlCommand} ${explorerUrl}/download/${distribId} --output export.zip
+                    ${curlCommand} ${nuxeoUrl}/site/automation/Elasticsearch.WaitForIndexing -H 'Content-Type:application/json+nxrequest' -X POST -d '{"params":{"timeoutSecond": "3600", "refresh": "true", "waitForAudit": "true"},"context":{}}'
```

Comment:
```suggestion
                    ${curlCommand} ${nuxeoUrl}/site/automation/Elasticsearch.WaitForIndexing -H 'Content-Type:application/json' -X POST -d '{"params":{"timeoutSecond": "3600", "refresh": "true", "waitForAudit": "true"},"context":{}}'
```
If I remember correctly the `+nxrequest` is not mandatory since 10.10 and should not be used.